articles/getting-started.md

```diff
@@ -0,0 +1,15 @@
+# Getting started
+
+The process to set up your system to be able to use the Bonsai.Harp package is simple, and documented in the next few steps:
+
+1 - Install Bonsai (if not already installed) by following the instructions available [here](https://bonsai-rx.org/docs/articles/installation.html).
+
+2 - Install the FTDI drivers necessary for the USB communication between Harp device and the host PC. For Windows, these can be found [here](https://www.ftdichip.com/old2020/Drivers/CDM/CDM%20v2.12.26%20WHQL%20Certified.zip).
+
+3 - Download the latest release of Bonsai.Harp package through the Bonsai Package Manager. [Detailed instructions can be found here](https://bonsai-rx.org/docs/articles/packages.html)
```

Comment:
I would not want to add more information in case, in the future, the package manager changes.. This way everything links to a common resource. I added in the next step because it deviates from the usual package source.
Yes I think we should add Harp.Design